.github/workflows/update-artifacts.yml

```diff
@@ -40,7 +40,10 @@ jobs:
       # - a polkadot.rs file from the full metadata that is checked in integration tests
       # - a polkadot.json in the `artifacts/demo_chain_specs` directory
       - name: Fetch Artifacts
-        run: cargo build --bin artifacts
+        run: cargo run --bin artifacts
+
+      - name: Delete substrate node binary
```

Comment:
My understanding is that this CI job:

- checks our branch
- downloads substrate-node binary to help make changes
- opens PR to capture any changes.

And since we don't want the `substrate-node` binary to be a part of this PR we delete it first

Reply:
It is exactly like James said, without removing the `substrate-node` it will be added to the PR like it happened today in the morning.